index.html

```diff
@@ -1218,37 +1218,67 @@ <h4>
+            </div>
           </ol>
         </li>
+        <li id="table-header-row-column">
+            <p its-locale-filter-list="en" lang="en">Arrangement of the table header row/column.</p>
+            <p its-locale-filter-list="zh-hans" lang="zh-hans">表格标题行、标题列的位置</p>
+            <p its-locale-filter-list="zh-hant" lang="zh-hant">表格標題行、標題列的位置</p>
+            <ol>
+              <li id="table-header-row-column-vertical">
+                <p its-locale-filter-list="en" lang="en" class="checkme">
+                  In vertical writing mode, put the header row of the table on the right and the header column on the top.
+                </p>
+                <p its-locale-filter-list="zh-hans" lang="zh-hans">直排时，表格的标题行（直行）位于右侧，标题列（横列）位于上方。</p>
+                <p its-locale-filter-list="zh-hant" lang="zh-hant">直排時，表格的標題行（直行）位於右側，標題列（橫列）位於上方。</p>
+              </li>
+              <li id="table-header-row-column-horizontal">
+                <p its-locale-filter-list="en" lang="en" class="checkme">
+                    In horizontal writing mode, put the header row of the table on the top and the header column on the left.
+                  </p>
+                  <p its-locale-filter-list="zh-hans" lang="zh-hans">横排时，表格的标题行（横行）位于上方，标题列（直列）位于左侧。</p>
+                  <p its-locale-filter-list="zh-hant" lang="zh-hant">橫排時，表格的標題行（橫行）位於上方，標題列（直列）位於左側。</p>
+              </li>
+              <div class="note" id="hang-lie">
+                <p its-locale-filter-list="en" lang="en" class="checkme">
+                    In different regions, the arrangement direction of cells described by <i>row</i> (Chinese: 行 <i>háng</i>) and <i>column</i> (Chinese: 列 <i>liè</i>) in Chinese terminology may be different. For example, 行 in mainland China usually refers to a horizontal arrangement of cells, and 列 refers to a vertical arrangement of cells; while 行 in Taiwan and Hong Kong usually refers to a vertical arrangement of cells, and 列 (or 栏) refers to a horizontal arrangement of cells.
+                  </p>
+                  <p its-locale-filter-list="zh-hans" lang="zh-hans">在不同的地区，中文术语中的「行」（háng）和「列」（liè）所描述的单元格排列方向，可能是不同的。例如，中国大陆的「行」通常指一组水平排列的单元格，「列」指一组垂直排列的单元格；而台湾、香港地区的「行」通常指一组垂直排列的单元格，「列」（或「栏」）指一组水平排列的单元格。</p>
+                  <p its-locale-filter-list="zh-hant" lang="zh-hant" class="checkme">在不同的地區，中文術語中的「行」（háng）和「列」（liè）所描述的單元格排列方向，可能是不同的。例如，中國大陸的「行」通常指一組水平排列的單元格，「列」指一組垂直排列的單元格；而臺灣、香港地區的「行」通常指一組垂直排列的單元格，「列」（或「欄」）指一組水平排列的單元格。</p>
+
+                  <p its-locale-filter-list="en" lang="en" class="checkme">
+                    This document uses row/column in logical directions consistently, i.e., "row" is vertical in vertical writing mode and horizontal in horizontal writing mode, and "column" is horizontal in vertical writing mode and vertical in horizontal writing mode.
+                  </p>
+                  <p its-locale-filter-list="zh-hans" lang="zh-hans">本文档采用「逻辑行」「逻辑列」来统一表述，也即，「行」在直排模式下垂直、在横排模式下水平，「列」在直排模式下水平、在横排模式下垂直。</p>
```

Comment:
Do we need to indicate the incrementing direction in here: e.g. 
> This document uses row/column in logical directions consistently, i.e., "row" is vertical going top-down in vertical writing mode and horizontal going left-right in horizontal writing mode, and "column" is horizontal going right-left in vertical writing mode and vertical going top-down in horizontal writing mode.
> 本文档采用「逻辑行」「逻辑列」来统一表述，也即，「行」在直排模式下上至下垂直、在横排模式下左至右水平，「列」在直排模式下右至左水平、在横排模式下上至下垂直。

The main reason is this edit did not reference back to W3C as suggested in https://github.com/w3c/clreq/issues/369#issuecomment-997433306, so the incrementing direction is not referenced. 
> CLReq 可以进一步考虑，是否需要对逻辑方向做出更详细的阐释。
If it is deemed ignorable then do ignore this comment.